src/main/resources/public/src/views/Dashboard.vue

```diff
@@ -1,13 +1,54 @@
 <template>
-    
+    <div id="container" class="container">
+        <h2>Dashboard</h2>
+        <table class="table table-sm">
+            <thead class="thead-dark">
+                <tr>
+                    <th>Nom</th>
+                    <th>DC Model</th>
+                    <th>Organization</th>
+                    <th>State</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr v-for="dataRequest in dataRequests">
+                    <td>{{ dataRequest.nom }}</td>
+                    <td>{{ dataRequest.model }}</td>
+                    <td>{{ dataRequest.organization }}</td>
+                    <td>{{ dataRequest.state }}</td>
+                </tr>
+            </tbody>
+        </table>
+    </div>
 </template>
 
 <script>
+    import axios from 'axios'
     export default {
-        name: "Dashboard"
+        name: "Dashboard",
+        data() {
+            return {
+              dataRequests: [],
+              errors: []
+            }
+        },
+        beforeCreate() {
+            axios.get(`api/data_access_request/123456789/sent`)
```

Comment:
is this a hardcoded URL ?

Reply:
yes, because the endpoint need siret and we don't have it in the interface, so temporarily I hardcod it